tests/src/test/scala/ch/renku/acceptancetests/HandsOnSpec.scala

```diff
@@ -29,6 +29,8 @@ class HandsOnSpec
 
     implicit val projectDetails: ProjectDetails = ProjectDetails.generateHandsOnProject(captureScreenshots)
 
+    implicit val docsScreenshots = new DocsScreenshots(this, captureScreenshots, browser)
```

Comment:
I should maybe ask about that in some previous PR but only noticed that now. It's about the `captureScreenshots` val. What do you think if the value is read from the args/env vars as part of the `DocsScreenshots` instantiation? I'm wondering if that wouldn't be maybe more suitable place for it instead of the `HandsOnSpecData`. I guess if you would like to generate screenshots for other flows the `DocsScreenshots` could be a bit simpler to use.

Reply:
The DocsScreenshots class was introduced in this PR, so your suggestion would not have been possible in the last PR.

This is a good suggestion! I have implemented it.